ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -319,10 +322,15 @@
       "The key store certificate alias to be used for internal client requests. If not set, "
           + "the system will fall back on the Vert.x default choice";
 
-  public static final String KSQL_LOGGING_SKIP_RESPONSE_CODES_CONFIG =
-      KSQL_CONFIG_PREFIX + "logging.skip.response.codes";
-  private static final String KSQL_LOGGING_SKIP_RESPONSE_CODES_DOC =
-      "A list of response codes to skip logging";
+  public static final String KSQL_LOGGING_SKIPPED_RESPONSE_CODES_CONFIG =
+      KSQL_CONFIG_PREFIX + "logging.skipped.response.codes";
```

Comment:
Can we make this `ksql.logging.server.skipped.response.codes` or similar to indicate that this is only for server request logging, in contrast to the `ksql.logging.processing.*` configs for the processing log? I considered suggesting `ksql.logging.request.*` rather than `ksql.logging.server.*`but I worry that a config such as `ksql.logging.request.skipped.response.codes` could be confusing. Maybe it's OK and that's actually better, unsure.

If we change the config name, we'll also have to update the docs. (Thanks for adding those!)